root_orchestrator/resource-abstractor/db/mongodb_client.py

```diff
@@ -0,0 +1,24 @@
+import os
+from flask_pymongo import PyMongo
+
+MONGO_URL = os.environ.get('CLOUD_MONGO_URL')
+MONGO_PORT = os.environ.get('CLOUD_MONGO_PORT')
+
+MONGO_ADDR_CLUSTERS = 'mongodb://' + str(MONGO_URL) + ':' + str(MONGO_PORT) + '/clusters'
+MONGO_ADDR_JOBS = 'mongodb://' + str(MONGO_URL) + ':' + str(MONGO_PORT) + '/jobs'
```

Comment:
I recommend using python's "f-strings" wherever possible. They are simpler and faster.

```suggestion
MONGO_ADDR_JOBS = f"mongodb:{MONGO_URL}:{MONGO_PORT}/jobs"
```

For more info see: https://www.geeksforgeeks.org/formatted-string-literals-f-strings-python/

Reply:
Agreed! I think I was following the pattern used in the repo, given also that this url is being used in another module.